coalib/results/Diff.py

```diff
@@ -375,12 +375,25 @@ def add_lines(self, line_nr_before, lines):
 
         linediff = self._get_change(line_nr_before, min_line=0)
         if linediff.add_after is not False:
-            raise ConflictError('Cannot add lines after the given line since '
-                                'there are already lines.')
+            raise ConflictError('Cannot add line(s) after the given line since '
+                                'there are already line(s).')
```

Comment:
I think using `line(s)` would work in both the cases. When a user gets a ConflictError on adding a single line, it is understood that user has added a single line.

Even if `add_lines` is used, and a single line in a `list` is passed, then also it will treat it as a multiple lines.
So using a generic statement would be fine.